python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -1524,6 +1586,42 @@ def _get_toolkit_manager(self):
         return self.TOOLKIT_MANAGER
 
     @sgtk.LogManager.log_timing
+    def _get_shotgun_yml_files(self, config_descriptor):
```

Comment:
A shallow dive into the config is still needed when building the list of entity types we want to provide actions for. That relies on the existence of any shotgun_xxx.yml environments that might exist, so a separate method is now provided to get only that list of files.